### PR TITLE
Clean up imports

### PR DESCRIPTION
**Pull request by gravesm**

### CODE REVIEW

File: magma/home/__init__.py [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+import tempfile
 from contextlib import closing
-try:
-    from StringIO import StringIO
-except ImportError:
-    from io import StringIO
+from zipfile import BadZipfile
 
-from flask import (Blueprint, flash, request, render_template, make_response,
-                   session,)
```
Comment (JPrevost) [inline magma/home/__init__.py:14]: Damn. I thought I confirmed this was necessary before committing. Sorry.

File: tests/test_views.py [hunk 2]
```diff
@@ -18,17 +17,20 @@ def test_submit_valid_form(testapp):
...
     res = form.submit()
     assert res.content_type == 'text/xml'
 
+
 def test_submit_without_shapefile(testapp):
     form = testapp.get('/').form
     res = form.submit()
     assert 'A datafile is required to proceed' in res
 
+
```
Comment (JPrevost) [inline tests/test_views.py:33]: I thought these should be single blank lines in pep8?
https://www.python.org/dev/peps/pep-0008/#blank-lines

File: magma/home/__init__.py (hunk 1 above)
Comment (gravesm) [inline magma/home/__init__.py:14]: Yeah, I didn't really notice that it wasn't used earlier, but my linter picked it up.

File: tests/test_views.py (hunk 2 above)
Comment (JPrevost) [inline tests/test_views.py:33]: Or is this double because they aren't in a class?

Comment (gravesm) [inline tests/test_views.py:33]: Yeah, methods are separated by a single space, but module-level functions should have two spaces between them.

Comment (JPrevost) [inline tests/test_views.py:33]: Thx. I've got pep8 linter working local now so hopefully I'll stop screwing that (  stupid ;)  ) rule up.

**Pull request merged**